### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by Mianto**
RegistrationHandler.class shows an error message when it is empty

### CODE REVIEW

File: java/source/org/libreoffice/ide/eclipse/java/JavaResourceDeltaVisitor.java [hunk 1]
```diff
@@ -110,7 +111,11 @@ private void removeImplementation(IResourceDelta pDelta,
             prjPath = prjPath.replace(".java", ""); //$NON-NLS-1$ //$NON-NLS-2$
             prjPath = prjPath.replace("/", "."); //$NON-NLS-1$ //$NON-NLS-2$
 
-            Vector<String> classes = RegistrationHelper.readClassesList(pUnoprj);
+            if RegistrationHelper.isFileEmpty(pUnoproj) {
+                PluginLogger.error(ResourceHelper.Messages.getString("RegistrationHelper.WriteClassesListError"));
```
Comment (smehrbrodt) [inline java/source/org/libreoffice/ide/eclipse/java/JavaResourceDeltaVisitor.java]: The error message talks about a write error, but this is a read error I guess?

File: java/source/org/libreoffice/ide/eclipse/java/JavaResourceDeltaVisitor.java [hunk 2]
```diff
@@ -128,7 +133,11 @@ private void removeImplementation(IResourceDelta pDelta,
     private void addImplementation(IResourceDelta pDelta, IUnoidlProject pUnoProject) {
         String className = isJavaServiceImpl(pDelta.getResource());
         if (className != null) {
-            RegistrationHelper.addImplementation(pUnoProject, className);
+            if RegistrationHelper.isFileEmpty(pUnoproj) {
+                PluginLogger.error(ResourceHelper.Messages.getString("RegistrationHelper.WriteClassesListError"));
```
Comment (smehrbrodt) [inline java/source/org/libreoffice/ide/eclipse/java/JavaResourceDeltaVisitor.java]: Same here.

File: core/doc/help/html/userdoc/javatuto/index.html [hunk 3]
```diff
@@ -185,26 +185,10 @@ <h4 class="title"><a name="id2473631"></a>Eclipse and the plugin</h4>
...
-<table border="0" summary="manufactured viewport for HTML img"
-	cellspacing="0" cellpadding="0" width="384">
-	<tr style="height: 244px">
-		<td><img src="img/img001.png" height="244"
-			alt=': "Find &amp; Install" window'></td>
-	</tr>
-</table>
-</div>
-</div>
+Eclipse using the <a href=https://marketplace.eclipse.org/content/loeclipse>Eclipse Marketplace</a>.<br/><br/>
```
Comment (smehrbrodt) [inline core/doc/help/html/userdoc/javatuto/index.html]: Please move this doc change in a separate pull request.

**Review by smehrbrodt — CHANGES_REQUESTED** [review]
Looks good in general, just note the comments I left inline.

**Comment by smehrbrodt** [comment]
Also please add a more descriptive PR title, like "Show error message when RegistrationHandler.classes is empty"

**Comment by Mianto** [comment]
Please review the changes now.Let me know if I had to do further changes.

**Comment by smehrbrodt** [comment]
Did you test this?

**Comment by Mianto** [comment]
Sorry I have not tested it. I will test it and commit changes.Sorry once again.

**Comment by Mianto** [comment]
Please review my changes now.

**Comment by smehrbrodt** [comment]
How did you test this? Were you able to launch an Eclipse instance with the LOEclipse plugin and verify that your fix works, as described in the [readme](https://github.com/LibreOffice/loeclipse#development)?

**Comment by smehrbrodt** [comment]
Superseded by #67.

**Pull request closed without merging**